wagtail/admin/templates/wagtailadmin/shared/_workflow_init.html

```diff
@@ -28,6 +28,7 @@
                                 e.currentTarget.click();
                             }
                         },
+                        'triggerElement': e.currentTarget,
```

Comment:
I wonder if there are other invocations of `ModalWorkflow` that would benefit from specifying `triggerElement`.

Reply:
I had a look through all other usage.

I cannot see any other cases where we try to re-click the element that was originally clicked & blocked so I think we are OK to keep this focused on the one usage fix for now.